content/03.categorize.md

```diff
@@ -231,6 +231,58 @@ a need for better pathologist-computer interaction techniques that will allow
+[@doi:10.1093/database/bax024] employed a maximum entropy model and a CNN model for chemical-induced disease relation extraction at inter- and intra-sentence level, respectively.
+For drug-drug interaction, [@doi:10.1093/bioinformatics/btw486] used a syntax CNN, [@doi:10.18653/v1/w17-2302] used an attention CNN, and [@arxiv:1705.03261] examined a recurrent neural network model with multiple attention layers.
+For biotopes event extraction, [@doi:10.18653/v1/w16-3012] employed CNN and distributed representation while [@doi:10.18653/v1/w16-3009] used long-short term memory (LSTM) networks to extract complicate relations.
+[@doi:10.18653/v1/w15-3814] applied word embedding to extract complete events from biomedical text and archived the results comparable to the state-of-the-art systems.
+There are also some works that identified event triggers rather than the complete event [@doi:10.1142/S0219720015410012; @arxiv:1705.09516].
+Taken together, it appears that deep learning models outperform traditional kernel methods or feature-based approaches by 1-5% in f-score.
+Among various deep learning approaches, CNN stands out as the most popular model both in terms of computational complexity and performance, while recurrent neural network (RNN) has achieved continuous progress.
+
+Information retrieval is a task of finding relevant text that satisfies an information need from within a large document collection.
+While deep learning has not yet achieved the same level of success in this area as seen in others, the recent surge of interest and work suggest that this may be quickly changing.
+For example, [@doi:10.18653/v1/w17-2328] described a deep learning approach to modeling the relevance of a document's text to a query, applied to the entire biomedical literature.
+
+In the clinical domain, [@pmid:27885364] employed a bidirectional LSTM RNN structure to extract adverse drug events from electronic health records.
+[@arxiv:1511.05942] developed Doctor AI, a RNN-based model that can learn efficient patient representation from a large amount of longitudinal patient records and predict diagnosis and medication code of EHR.
+[@doi:10.3233/978-1-61499-432-9-584] applied the word2vec [@arxiv:1301.3781] deep learning toolkit to medical corpora and evaluated the efficiency of word2vec in identifying properties of pharmaceuticals based on mid-sized, unstructured medical text corpora without any additional background knowledge.
+[@doi:10.18653/v1/w17-2341] investigated using CNN to extract temporal relations.
+[@doi:10.18653/v1/w17-2342] investigated the applicability of deep learning at autocoding of radiology reports using the International Classification of Diseases [@tag:world2004international].
+
+To summarize, deep learning has shown promising results in many biomedical text mining tasks and applications.
+But to further realize its potential, more technical advances and/or labeled data would be required.
```

Comment:
If we expand this lightly to:
>But to further realize its potential in this domain, either large corpora of labeled data and/or technical advances that use the limited labeled available data more efficiently are required.
Is that accurate? If so, can you use this to give people a bit more of the summary in the final sentence?